org.opent2t.sample.lamp.superpopular/com.hue.bulb/js/manifest.xml

```diff
@@ -5,10 +5,14 @@
   <!-- associated schemas -->
   <schemas>
     <schema id="org.opent2t.sample.lamp.superpopular" main="true" type="OCF" />
+    <schema id="opent2t.platform" type="OCF" />
     <schema id="oic.core" type="OCF" />
     <schema id="oic.baseresource" type="OCF" />
+    <schema id="oic.oic-link" type="OCF" />
     <schema id="oic.r.switch.binary" type="OCF" />
     <schema id="oic.r.light.dimming" type="OCF" />
     <schema id="oic.r.colour.rgb" type="OCF" />
+    <schema id="oic.r.colour.chroma" type="OCF" />
```

Comment:
This is the OIC resource.  OCF uses the British (?) spelling ([https://openconnectivity.org/specs/OIC_1.1-Specification.zip](https://openconnectivity.org/specs/OIC_1.1-Specification.zip) see OIC_Resource_Type_Specification_v1.1.0.pdf)

---
In reply to: [94472847](https://github.com/openT2T/translators/pull/143#discussion_r94472847) [](ancestors = 94472847)